components/button/src/index.js

```diff
@@ -38,8 +39,15 @@ const StyledButton = styled('button')(
     textDecoration: 'none',
     WebkitAppearance: 'none',
     WebkitFontSmoothing: 'antialiased',
+    ':hover': {
+      backgroundColor: GREEN,
+    },
     ':focus': {
+      position: 'absolute',
+      top: '2px',
+      backgroundColor: GREEN,
       outline: `3px solid ${YELLOW}`,
+      boxShadow: 'none',
```

Comment:
Instead of setting it to `none`, I think it could be better for browsers to adjust the value to `0px` (could have a potential performance impact).